__tests__/app.test.js

```diff
@@ -15,3 +15,23 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects, each having slug and description ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toBeInstanceOf(Array);
```

Comment:
As you are checking the length and then using an array method (the forEach) in the next 2 assertions this assertion is not required/redundant.